scanner/src/main/kotlin/Scanner.kt

```diff
@@ -144,22 +140,19 @@ class Scanner(
             }
         }
 
-        val scannerRun = ScannerRun(
-            startTime = startTime,
-            endTime = endTime,
-            environment = Environment(toolVersions = toolVersions),
-            config = scannerConfig,
-            provenances = projectResults.provenances + packageResults.provenances,
-            scanResults = projectResults.scanResults + packageResults.scanResults,
-            issues = projectResults.issues + packageResults.issues,
-            files = projectResults.files + packageResults.files,
-            scanners = projectResults.scanners + packageResults.scanners
+        val scannerRun = (projectResults + packageResults).copy(
+            environment = Environment(toolVersions = toolVersions)
         )
 
-        return ortResult.copy(scanner = scannerRun)
+        val paddedScannerRun = scannerRun.takeUnless { scannerConfig.includeFilesWithoutFindings }
+            ?: scannerRun.padNoneLicenseFindings()
+
+        return ortResult.copy(scanner = paddedScannerRun)
     }
 
     internal suspend fun scan(packages: Set<Package>, context: ScanContext): ScannerRun {
+        val startTime = Instant.now()
+
```

Comment:
Let's move this a bit down so it's initialized if we return early in line 159. I propose to move this block to current line 166, so it marks the start of the block of function calls there.